benchmarks/build.sbt

```diff
@@ -16,12 +14,8 @@ Compile / doc / sources ~= (_ filter (_ => false))
 
 // we want to consume this from a java8 build
 compile / javacOptions ++= Seq("--release", "8")
-scalacOptions ++= Seq() ++ (
-  CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((3, _)) => Seq("-Xtarget:8")
-    case _            => Seq("-target:jvm-1.8")
-  }
-)
+
+scalacOptions += "-Xtarget:8"
```

Comment:
Cant we set this for all subprojects in the main `build.sbt`?

Reply:
good idea, done